### DISCUSSION TITLE
What are the size limits on websites using the free plan?

### DISCUSSION THREAD

**Discussion by Josrph** [vercel/community#1431] [Help] [answered]
What are the size limits on websites using the free plan? also will the site be up 24/7?

**Comment by swarnava** [comment, marked as answer]
Couple of points:

- Hobby plan provide 13 gb Disk Size and 8gb of RAM. You can learn rest of limit here: https://vercel.com/docs/concepts/limits/overview
- Hobby plan only meant for Personal project. For commercial usecase, please consider paid plan: https://vercel.com/docs/concepts/limits/fair-use-policy
- Hobby projects will remain up for 24/7 unless it's violate our Terms & Condition or Fair Usage Policy.

**Reply by Josrph** [reply]
Ok, more question, Is there any limits on the number of projects you deploy? also the bandwidth limit (100 GB) is for all projects or for each project separately?

**Reply by swarnava** [reply]
There is no such a limit. Bandwith limit is across all projects in particular scope/account/team

**Reply by Josrph** [reply]
One last question, from reading the limits page, there is a lot about Serverless Functions, but what if I am using express as my backend, how will the limits apply?

**Reply by swarnava** [reply]
If you aren't using Serverless function, it won't be counted against your usage

**Reply by Josrph** [reply]
Perfect, Thank you so much!

**Reply by leoj3n** [reply]
@swarnava Can you clarify what is meant by "Disk Size" and/or "Disk space"?

The linked docs say _"Every Build is provided with the following resources: Disk space 13 GB"_ as well as _"we apply the following limits to all accounts. Disk Size (GB) | 13"_.

I assume in both instances the 13GB limit is size on disk at build time of any given project, and not a limit across all projects as that would seem too restrictive if you have many projects with lots of image assets per project for instance.

Is there any limit on how large an imported Git repository can be? I see an account can have 200 Projects for instance, does this mean that theoretically each project could potentially have 12 GB of assets per project, but would trigger the limit if the built output > 13?

It's just not clear if 13 GB Disk Size/Space is in regards to the size of a Project repo, size while building, or the size after a build... And if this applies "per Project", or is a cumulative limit across all Projects. It's not clear what would trigger this limit... For example, would a large number of high resolution images in the repo that are also included in the built output be a trigger?

Looking under Usage for a Team I don't see a percentage used of this 13 GB... Searching for "Disk" on the Usage page shows 0 results.
